catalog/content/content-app/src/main/resources/features.xml

```diff
@@ -57,12 +57,17 @@
              description="REST Endpoint provides CRUD operations for content storage.">
         <bundle>mvn:ddf.content.endpoint/content-rest-endpoint/${project.version}</bundle>
     </feature>
-    
+
     <feature name="content-catalogerplugin" install="manual" version="${project.version}"
              description="Content Plugin provides post-CUD operations after content storage.">
         <bundle>mvn:ddf.content.core/content-core-catalogerplugin/${project.version}</bundle>
     </feature>
-    
+
+    <feature name="content-core-videothumbnailplugin" install="manual" version="${project.version}"
```

Comment:
I am debating if this feature should be installed with the content app.  I want to say yes since this is unlikely to break anything.  Plus we have very limited video support right now anyways.